recipe_sha: 95c8dd202b5002f712ebb93b42c0a841958c188267059bc46cf5b75dd60613e7  # The hash of the recipe's meta.yaml at which this recipe failed to build.
skiplist: false # Set to true to skiplist this recipe so that it will be ignored as long as its latest commit is the one given above.
log: |
  Adding in variants from internal_defaults
  Adding in variants from /opt/host-conda-bld/conda_build_config_0_-e_conda_build_config.yaml
  Adding in variants from /opt/host-conda-bld/conda_build_config_1_-e_bioconda_utils-conda_build_config.yaml
  Skipped: pywfa from /opt/recipe defines build/skip for this configuration ({'c_compiler_version': '12', 'target_platform': 'linux-64', 'CONDA_BUILD_SYSROOT': '', 'python': '3.10.* *_cpython', 'c_compiler': 'gcc', 'channel_targets': 'bioconda main'}).
  Renaming work directory '/opt/conda/conda-bld/pywfa_1717538405009/work' to '/opt/conda/conda-bld/pywfa_1717538405009/work_moved_pywfa-0.5.1-py310h7c593f9_3_linux-64_main_build_loop'
  shutil.move(work)=/opt/conda/conda-bld/pywfa_1717538405009/work, dest=/opt/conda/conda-bld/pywfa_1717538405009/work_moved_pywfa-0.5.1-py310h7c593f9_3_linux-64_main_build_loop)
  Skipped: pywfa from /opt/recipe defines build/skip for this configuration ({'c_compiler_version': '12', 'target_platform': 'linux-64', 'CONDA_BUILD_SYSROOT': '', 'python': '3.12.* *_cpython', 'c_compiler': 'gcc', 'channel_targets': 'bioconda main'}).
  Renaming work directory '/opt/conda/conda-bld/pywfa_1717538405009/work' to '/opt/conda/conda-bld/pywfa_1717538405009/work_moved_pywfa-0.5.1-py312hf67a6ed_3_linux-64_main_build_loop'
  shutil.move(work)=/opt/conda/conda-bld/pywfa_1717538405009/work, dest=/opt/conda/conda-bld/pywfa_1717538405009/work_moved_pywfa-0.5.1-py312hf67a6ed_3_linux-64_main_build_loop)
  Skipped: pywfa from /opt/recipe defines build/skip for this configuration ({'c_compiler_version': '12', 'target_platform': 'linux-64', 'CONDA_BUILD_SYSROOT': '', 'python': '3.11.* *_cpython', 'c_compiler': 'gcc', 'channel_targets': 'bioconda main'}).
  Renaming work directory '/opt/conda/conda-bld/pywfa_1717538405009/work' to '/opt/conda/conda-bld/pywfa_1717538405009/work_moved_pywfa-0.5.1-py311hdad781d_3_linux-64_main_build_loop'
  shutil.move(work)=/opt/conda/conda-bld/pywfa_1717538405009/work, dest=/opt/conda/conda-bld/pywfa_1717538405009/work_moved_pywfa-0.5.1-py311hdad781d_3_linux-64_main_build_loop)
  Attempting to finalize metadata for pywfa
  Reloading output folder (file:///opt/host-conda-bld): ...working... done
  Getting pinned dependencies: ...working... done
  Reloading output folder (file:///opt/host-conda-bld): ...working... done
  Getting pinned dependencies: ...working... done
  Reloading output folder (file:///opt/host-conda-bld): ...working... done
  Getting pinned dependencies: ...working... done
  BUILD START: ['pywfa-0.5.1-py39hff71179_3.tar.bz2']
  Reloading output folder (file:///opt/host-conda-bld): ...working... done
  Solving environment (_h_env): ...working... done

  ## Package Plan ##

    environment location: /opt/conda/conda-bld/pywfa_1717538405009/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho


  The following NEW packages will be INSTALLED:

      _libgcc_mutex:    0.1-conda_forge           conda-forge
      _openmp_mutex:    4.5-2_gnu                 conda-forge
      bzip2:            1.0.8-hd590300_5          conda-forge
      ca-certificates:  2024.6.2-hbcca054_0       conda-forge
      cython:           3.0.10-py39h3d6467e_0     conda-forge
      ld_impl_linux-64: 2.40-hf3520f5_2           conda-forge
      libffi:           3.4.2-h7f98852_5          conda-forge
      libgcc-ng:        13.2.0-h77fa898_7         conda-forge
      libgomp:          13.2.0-h77fa898_7         conda-forge
      libnsl:           2.0.1-hd590300_0          conda-forge
      libsqlite:        3.45.3-h2797004_0         conda-forge
      libstdcxx-ng:     13.2.0-hc0a3c3a_7         conda-forge
      libuuid:          2.38.1-h0b41bf4_0         conda-forge
      libxcrypt:        4.4.36-hd590300_1         conda-forge
      libzlib:          1.3.1-h4ab18f5_1          conda-forge
      ncurses:          6.5-h59595ed_0            conda-forge
      openssl:          3.3.0-h4ab18f5_3          conda-forge
      pip:              24.0-pyhd8ed1ab_0         conda-forge
      python:           3.9.19-h0755675_0_cpython conda-forge
      python_abi:       3.9-4_cp39                conda-forge
      readline:         8.2-h8228510_1            conda-forge
      setuptools:       70.0.0-pyhd8ed1ab_0       conda-forge
      tk:               8.6.13-noxft_h4845f30_101 conda-forge
      tzdata:           2024a-h0c530f3_0          conda-forge
      wheel:            0.43.0-pyhd8ed1ab_1       conda-forge
      xz:               5.2.6-h166bdaf_0          conda-forge

  Preparing transaction: ...working... done
  Verifying transaction: ...working... done
  Executing transaction: ...working... done
  Rolling back transaction: ...working... done
  Traceback (most recent call last):
    File "/opt/conda/bin/conda-build", line 11, in <module>
      sys.exit(execute())
    File "/opt/conda/lib/python3.10/site-packages/conda_build/cli/main_build.py", line 590, in execute
      api.build(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/api.py", line 250, in build
      return build_tree(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 3638, in build_tree
      packages_from_this = build(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 2409, in build
      create_build_envs(top_level_pkg, notest)
    File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 2183, in create_build_envs
      environ.create_env(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/environ.py", line 1049, in create_env
      _execute_actions(prefix, precs)
    File "/opt/conda/lib/python3.10/site-packages/conda_build/environ.py", line 1334, in _execute_actions
      unlink_link_transaction.execute()
    File "/opt/conda/lib/python3.10/site-packages/conda/core/link.py", line 354, in execute
      self._execute(
    File "/opt/conda/lib/python3.10/site-packages/conda/core/link.py", line 970, in _execute
      raise CondaMultiError(
  conda.CondaMultiError: [Errno 28] No space left on device
  ()










  The target environment has been corrupted. Corrupted environments most commonly
  occur when the conda process is force-terminated while in an unlink-link
  transaction.
    environment location: /opt/conda/conda-bld/pywfa_1717538405009/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho
    corrupted file: /opt/conda/conda-bld/pywfa_1717538405009/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/conda-meta/ncurses-6.5-h59595ed_0.json
# Last 100 lines of the build log.
